### PR TITLE
update scala version and dependencies

### PR DESCRIPTION
**Pull request by LevT2**

### CODE REVIEW

File: .bsp/sbt.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"name":"sbt","version":"1.5.5","bspVersion":"2.0.0-M5","languages":["scala"],"argv":["C:\\A\\SCOOP\\apps\\adoptopenjdk-hotspot\\current/bin/java","-Xms100m","-Xmx100m","-classpath","C:/Users/admctv/AppData/Roaming/JetBrains/IdeaIC2021.1/plugins/Scala/launcher/sbt-launch.jar","xsbt.boot.Boot","-bsp","--sbt-launch-jar=C:/Users/admctv/AppData/Roaming/JetBrains/IdeaIC2021.1/plugins/Scala/launcher/sbt-launch.jar"]}
```
Comment (winitzki) [inline .bsp/sbt.json]: Please remove this file or add `.bsp` to `.gitignore`. This is autogenerated by newer IntelliJ.

File: build.sbt [hunk 2]
```diff
@@ -1,71 +1,74 @@
 lazy val common = Seq(
-  scalaVersion := "2.12.6",
+  //  scalaVersion := "2.12.6",
+  scalaVersion := "2.13.6",
   crossScalaVersions := Seq("2.11.12", "2.12.6"),
```
Comment (winitzki) [inline build.sbt]: `crossScalaVersions` should be probably changed to `Seq("2.11.12", "2.12.12", "2.13.6")`

Comment (LevT2) [reply build.sbt]: Done

**Comment by winitzki** [comment]
I will need to think about how to minimize changes. There are too many changes due to upgrading test libraries and so on.

**Comment by winitzki** [comment]
Work has been done and merged. curryhoward 0.3.8 supports Scala 2.13 as well as 2.11 and 2.12.

**Pull request closed without merging**